### PR TITLE
feat: setup script for .env configuration file

### LINKED ISSUE #1233: Create a Setup Script for the `.env` configuration file
**Is your feature request related to a problem? Please describe.**

1. The current `INSTALLATION.md` file describes how to manually setup the `.env` file
    1. https://github.com/PalisadoesFoundation/talawa-admin/blob/main/INSTALLATION.md#creating-env-file 
1. We need a setup script similar to that used in Talawa API that automates the population of the `.env` file

**Describe the solution you'd like**
1. We need a setup script similar to that used in Talawa API that automates the population of the `.env` file
1. It should validate the parameters entered and that any services related to the parameters are working correctly
1. The `INSTALLATION.md` file must be updated with the new procedure
1. The table of contents must be regenerated with a link to the new instructions visible

**Describe alternatives you've considered**
- N/A

**Approach to be followed (optional)**
- N/A

**Additional context**
- N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by disha1202**
**What kind of change does this PR introduce?**

**Issue Number:**

Fixes #1233 

**Did you add tests for your changes?**

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by disha1202** [comment]
@palisadoes 

Please take a look at this PR.

**Comment by disha1202** [comment]
@palisadoes 

Please refer to this comment https://github.com/PalisadoesFoundation/talawa-admin/pull/1459#issuecomment-1902230369

File: setup.ts [hunk 1]
```diff
@@ -0,0 +1,196 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```
Comment (palisadoes) [inline setup.ts:1]: 1. Please remove. Fix the cause of the linting error.
1. We need our linting rules to be followed. We have created an issue to enforce this as part of future PR workflows

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

**Comment by palisadoes** [comment]
1. This has not been happening with other PRs
2. The push workflow only applies when changes are made to the repo branches. PRs don't apply
3. It's also strange that the github actions are not being triggered.

Please make the suggested changes and let's get this PR merged.

File: setup.ts (hunk 1 above)
Comment (disha1202) [reply setup.ts]: Removed

**Review by palisadoes — APPROVED** [review]

**Comment by palisadoes** [comment]
The reason for the files being submitted was that you hadn't synced your branch with the latest upstream that had applied all those changes already.

**Pull request merged**